common/scripts/changelog/check.mjs

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/**
+ * Script to check that changefile has been generated for each Pull Request.
+ *
+ * This script is only useful in the context of PR validation.
+ *
+ * Usage:
+ *   node common/scripts/changelog/check.mjs origin/main origin/feature-pony
+ */
+
+import { REPO_ROOT } from "../lib/constants.mjs";
+import { exec_output } from "../lib/exec.mjs";
+import path from 'path';
+import { parseNewChangeFiles } from "./utils.mjs";
+
+async function main() {
+  const [base, head] = parseArgs(process.argv);
+  const gitLogStdout = await exec_output(`git log --name-status ${base}..${head} -- ${path.join(REPO_ROOT, 'change/')}`);
+  const newChangeFiles = parseNewChangeFiles(gitLogStdout);
+  if (!newChangeFiles?.length) {
+    console.error('No changefile detected! Please run `rush changelog` to document your change.');
+    process.exit(1);
+  }
+  console.log(`Found ${newChangeFiles.length} changefiles. All is good!`)
```

Comment:
Looks like we don't check changelog for each packages like `rush changelog:check` anymore, we just check overall number of change files here?

Reply:
Yes, we only maintain change-files for communication-react after this change.

Reply:
Read the doc, looks like we are going to generate entities only for main package, which makes a lot of sense